src/parsing/ris_live.go

```diff
@@ -0,0 +1,46 @@
+package parsing
+
+import (
+	"BGPAlert/common"
+	"encoding/json"
+	"fmt"
+	"log"
+
+	"github.com/gorilla/websocket"
+)
+
+func main() {
+	// RIS Live websocket url
+	socketUrl := "ws://ris-live.ripe.net/v1/ws/"
+
+	// create websocket connection to ris live websocket
+	conn, _, err := websocket.DefaultDialer.Dial(socketUrl, nil)
+	if err != nil {
+		log.Fatal("Websocket connection error:", err)
```

Comment:
Should you also exit the program?

Reply:
agreed - no point in continuing program execution without a connection being established.